pandas-stubs/core/reshape/pivot.pyi

```diff
@@ -1,36 +1,130 @@
+import datetime
 from typing import (
     Callable,
+    Hashable,
     Literal,
+    Mapping,
     Sequence,
+    TypeVar,
+    Union,
+    overload,
 )
 
+import numpy as np
+import pandas as pd
 from pandas.core.frame import DataFrame
 from pandas.core.groupby.grouper import Grouper
+from pandas.core.indexes.base import Index
 from pandas.core.series import Series
+from typing_extensions import TypeAlias
 
 from pandas._typing import (
-    IndexLabel,
+    Label,
     Scalar,
+    npt,
 )
 
+_HashableT1 = TypeVar("_HashableT1", bound=Hashable)
+_HashableT2 = TypeVar("_HashableT2", bound=Hashable)
+_HashableT3 = TypeVar("_HashableT3", bound=Hashable)
```

Comment:
Now you can use the ones in `_typing.pyi`

Reply:
Fixed.